description: jumbo build has trouble with the namespace of SnapContainerData
author: Michael Gilbert <mgilbert@debian.org>

--- a/third_party/blink/renderer/platform/graphics/paint/scroll_paint_property_node.h
+++ b/third_party/blink/renderer/platform/graphics/paint/scroll_paint_property_node.h
@@ -14,7 +14,7 @@
 #include "third_party/blink/renderer/platform/platform_export.h"
 #include "third_party/blink/renderer/platform/scroll/main_thread_scrolling_reason.h"
 #include "third_party/blink/renderer/platform/scroll/overscroll_behavior.h"
-#include "third_party/blink/renderer/platform/scroll/scroll_snap_data.h"
+#include "cc/input/scroll_snap_data.h"
 
 namespace blink {
 
@@ -52,7 +52,7 @@ class PLATFORM_EXPORT ScrollPaintPropert
     CompositorElementId compositor_element_id;
     OverscrollBehavior overscroll_behavior = blink::OverscrollBehavior(
         blink::OverscrollBehavior::kOverscrollBehaviorTypeAuto);
-    base::Optional<SnapContainerData> snap_container_data;
+    base::Optional<cc::SnapContainerData> snap_container_data;
 
     bool operator==(const State& o) const {
       return container_rect == o.container_rect &&
@@ -99,7 +99,7 @@ class PLATFORM_EXPORT ScrollPaintPropert
     return state_.overscroll_behavior.y;
   }
 
-  base::Optional<SnapContainerData> SnapContainerData() const {
+  base::Optional<cc::SnapContainerData> SnapContainerData() const {
     return state_.snap_container_data;
   }
 
